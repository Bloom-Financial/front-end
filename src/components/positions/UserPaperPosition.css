.container {
  height: 19vh;
  width: 5vw;
  background-color: green;
  color: #fff;
  border: solid 2px black;
  margin: 4px;
  position: relative;
  right: -75vw;
  color: aliceblue;
}
.symbol {
  font-size: 0.5rem;
}

.qty {
  font-size: 0.5rem;
}

.value {
  font-size: 0.5rem;
}
