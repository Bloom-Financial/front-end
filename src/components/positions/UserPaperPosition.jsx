import React from 'react';
import PropTypes from 'prop-types';
import styles from './UserPaperPosition.css';

const UserPaperPosition = ({ symbol, qty, market_value }) => (
  <dd className={styles.container}>
    <h1 className={styles.symbol}>{symbol}</h1>
    <h2>{qty}</h2>
    <h2>{market_value}</h2>
  </dd>
);

export default UserPaperPosition;

UserPaperPosition.propTypes = {
  symbol: PropTypes.string.isRequired,
  qty: PropTypes.string.isRequired,
  market_value: PropTypes.string.isRequired,
};
