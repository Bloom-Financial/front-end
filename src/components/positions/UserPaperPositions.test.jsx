import React from 'react';
import { render } from '@testing-library/react';
import UserPaperPosition from './UserPaperPosition';



describe('Test the user paper positions component', () => {
  it('tests the user paper positions component', () => {
    const { asFragment } = render(<UserPaperPosition/>);
    expect(asFragment()).toMatchSnapshot();
  });
});
