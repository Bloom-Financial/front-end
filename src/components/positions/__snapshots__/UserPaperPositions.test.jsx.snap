// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the user paper positions component tests the user paper positions component 1`] = `
<DocumentFragment>
  <dd
    class="container"
  >
    <h1
      class="symbol"
    >
      Stock: 
    </h1>
    <h2
      class="qty"
    >
      Qty: 
    </h2>
    <h2
      class="value"
    >
      Market Value:
    </h2>
    <h2
      class="price"
    >
      Current Price:
    </h2>
  </dd>
</DocumentFragment>
`;
